package.json

```diff
@@ -30,6 +30,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
+    "@types/semver": "^6.0.2",
```

Comment:
`semver` is a dependency, and therefore `@types/semver` should also be a dependency (not a devDependency). This matters for those installing esm-devtools as a dependency of their own package.